osmaxx/excerptexport/static/excerptexport/scripts/maps/draw_controls.js

```diff
@@ -82,6 +89,41 @@ var draw_controls = function (map) {
             nameField.parentNode.appendChild(e);
         }
 
+        function getAllowedMaxSize(){
+            var selectedDetailLevel = jQuery('#id_detail_level');
+            var selectedValue = selectedDetailLevel.find(":selected").attr('value');
+            if (parseInt(selectedValue) === 60) {
```

Comment:
As we've consciously chosen numerical choice values, would `<=` instead of `===` be appropriate?